mautrix_telegram/db/portal.py

```diff
@@ -53,6 +53,11 @@ def get_by_tgid(cls, tgid: TelegramID, tg_receiver: TelegramID) -> Optional['Por
     def find_private_chats(cls, tg_receiver: TelegramID) -> Iterable['Portal']:
         yield from cls._select_all(cls.c.tg_receiver == tg_receiver, cls.c.peer_type == "user")
 
+    @classmethod
+    def count(cls) -> int:
+        count = cls.db.execute(select([func.count('*')]).select_from(Table("portal", MetaData()))).scalar()
```

Comment:
I think this might work

```suggestion
        count = cls.db.execute(cls.t.select([func.count('*')])).scalar()
```